org.flightgear.FlightGear.json

```diff
@@ -0,0 +1,193 @@
+{
+	"id":"org.flightgear.FlightGear",
```

Comment:
nitpick but single space between keys and values please.

Reply:
Will do